src/main/java/net/openhft/chronicle/wire/Event.java

```diff
@@ -68,15 +68,9 @@ default E eventTimeNow() {
     }
 
     /**
-     * Updates event with new event name, updating event time to now if required.
-     *
-     * @param eventName name of the event
+     * Updates event with event time to now if required.
      */
-    // TODO: x.23 remove eventName parameter
-    default E updateEvent(final String eventName) {
-        if (this.eventId().length() == 0)
-            this.eventId(eventName);
-
+    default E updateEvent() {
```

Comment:
ah actually I don't think we should do this any more. We may undeprecate eventName

Reply:
great I was feeling unease about this one )